public/components/FeatureAnywhereContextMenu/CreateAnomalyDetector/helpers.tsx

```diff
@@ -30,10 +30,10 @@ export function visFeatureListToFormik(
 }
 
 export function formikToDetectorName(title) {
-  const detectorName =
+  let detectorName =
```

Comment:
nit: what is the reason of changing this to `let` from `const`?

Reply:
no specific reason, the constant variable vs mutable variable difference doesn't matter here. I was just playing around trying to figure out the constant not getting reassigned issue, didn't change it back after making it work. Changed it back to const